legal-api/src/legal_api/models/filing.py

```diff
@@ -110,8 +110,6 @@ def payment_token(self):
 
     @payment_token.setter
     def payment_token(self, token: int):
```

Comment:
as discussed, turn this back on and have the reset method alter the self._payment_token

Reply:
Done :)